docs/integrations/sources/postgres.md

```diff
@@ -81,18 +11,24 @@ If you can't find the data type you are looking for or have any problems feel fr
 | Logical Replication \(WAL\) | Yes |  |
 | SSL Support | Yes |  |
 | SSH Tunnel Connection | Coming soon |  |
```

Comment:
This is not supported now?

Reply:
It is! It just wasn't updated along with the addition of the SSH tunnel docs, updating now, thanks :)